src/ui/MultiUserChatWindow.ts

```diff
@@ -178,6 +275,173 @@ export default class MultiUserChatWindow extends ChatWindow {
       )
    }
 
+   protected onInputCommand(message)
+   {
+       console.log(this.contact);
+       let command = message.substring(1,message.indexOf(' ')!==-1?message.indexOf(' '):message.length).toLowerCase();
+       if (command === Command[Command.help])
```

Comment:
I think if we use a similar structure like in https://github.com/jsxc/jsxc/blob/master/src/Notice.ts the code could be a lot more readable.

Reply:
as i said im not the typescript pro... so if you would merge and change it to your style then?